clients/admin-ui/src/home/HomeContent.tsx

```diff
@@ -0,0 +1,119 @@
+import { Center, Flex, SimpleGrid, Text } from "@fidesui/react";
+import Link from "next/link";
+import * as React from "react";
+import { useEffect, useState } from "react";
+
+import { useFeatures } from "~/features/common/features.slice";
+
+import { MODULE_CARD_ITEMS, ModuleCardKeys } from "./constants";
+import { ModuleCard } from "./types";
+
+const HomeContent: React.FC = () => {
+  const COLUMNS = 3;
+  const [list, setList] = useState<ModuleCard[]>(
```

Comment:
This doesn't need to be set on the state, right? It'll only change due to the useEffect firing, which will happen when one of `[hasConnections, hasSystems]` changed and triggered a re-render. So creating the list from a `useMemo` would do the trick instead.

Reply:
Issue has been resolved.